python/tvm/relay/analysis/analysis.py

```diff
@@ -433,8 +433,7 @@ def get_calibration_data(mod, data):
     mod = _ffi_api.get_calibrate_module(mod)
     mod = transform.Inline()(mod)
 
-    ref_ex = build_module.create_executor("graph", mod=mod, device=cpu(0))
-    ref_res = ref_ex.evaluate()(**data)
+    ref_res = build_module.create_executor("graph", mod=mod, device=cpu(0)).evaluate()(**data)
```

Comment:
This is probably follow up, but this evaluate interface is sort of a hold over from the old days, we could probably just simplify the executor interface in another pass.

Reply:
Agree, and I'm very close to doing it now since I went and rejigged **all** the create_executor/evaluate calls anyway.